scaleway/validate_cassettes_test.go

```diff
@@ -59,6 +59,25 @@ func exceptionsCassettesCases() map[string]struct{} {
 	}
 }
 
+func checkErrCodeExcept(i *cassette.Interaction, c *cassette.Cassette, codes ...int) bool {
+	exceptions := exceptionsCassettesCases()
+	_, isException := exceptions[c.File]
+	if isException {
+		return isException
+	}
+
+	if i.Code >= 400 {
+		for _, httpCode := range codes {
+			if i.Code == httpCode {
+				return true
+			}
+		}
+		return false
+	}
+
+	return true
+}
+
 func checkErrCode(i *cassette.Interaction, c *cassette.Cassette, codes ...int) bool {
```

Comment:
Could you remove this function if it is not used anymore? or find another usage for it.